.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,29 +4,25 @@
 At a high level, describe what this PR does.
 -->
 
-### Types of changes:
+### Upgrade instructions
 
-- [ ] Bugfix
-- [ ] New feature
-- [ ] Refactor
-
-### Component:
+<!--
+If this PR makes changes that other people should be aware of when upgrading
+their code, describe what they should do between the two comments below.
```

Comment:
Many people don't know what an HTML comment looks like.  Maybe "between the two lines", or, to be explicit, "between the two UPGRADE INSTRUCTIONS lines".